.github/workflows/main.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: curl -L https://raw.githubusercontent.com/foundry-rs/starknet-foundry/master/scripts/install.sh | sh
 
       - name: Install snforge
-        run: snfoundryup -v 0.27.0
+        run: snfoundryup -v 0.30.0
```

Comment:
Hey @bitfalt just a reminder you might need to resync with `main` branch since we use newer snfoundry and scarb. Thanks in advance!

Reply:
snfoundry - 0.31.0
scarb - 2.8.3

Reply:
ohh okay, thanks for reminding :D